### PR TITLE
Implement ItemTappedEventArgsConverter and ConvertersGalleryPage

### PR DESCRIPTION
**Pull request by almirvuk**
### Description of Change ###

This is a very simple converter that allows you to extract `Item` value from `ItemTappedEventArgs` object and it can be used in combination with `EventToCommandBehavior`.

Added: 
 
- `ItemTappedEventArgsConverter`

### Sample

```
<c:ItemTappedEventArgsConverter x:Key="ItemTappedEventArgsConverter" />

<ListView.Behaviors>
      <b:EventToCommandBehavior EventName="ItemTapped" 
                                Command="{Binding ItemTappedCommand}"
                                EventArgsConverter="{StaticResource ItemTappedEventArgsConverter}" />
</ListView.Behaviors>   
```

### PR Checklist ###

- [x] Has tests (if omitted, state reason in description)
- [x] Has samples (if omitted, state reason in description)
- [x] Rebased on top of master at time of PR
- [x] Changes adhere to coding standard
- [ ] Updated documentation

### CODE REVIEW

File: XamarinCommunityToolkit/Converters/ItemTappedEventArgsConverter.shared.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        {
+            if (!(value is ItemTappedEventArgs itemTappedEventArgs))
+            {
+                throw new ArgumentException("Expected value to be of type ItemTappedEventArgs", nameof(value));
+            }
+
+            return itemTappedEventArgs.Item;
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) => throw new NotImplementedException();
```
Comment (AndreiMisiukevich) [inline XamarinCommunityToolkit/Converters/ItemTappedEventArgsConverter.shared.cs:31]:
```suggestion
            if (!(value is ItemTappedEventArgs itemTappedEventArgs))
            {
                throw new ArgumentException("Expected value to be of type ItemTappedEventArgs", nameof(value));
            }

            return itemTappedEventArgs.Item;
        }

        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) 
            => throw new NotImplementedException();
```

Comment (AndreiMisiukevich) [inline XamarinCommunityToolkit/Converters/ItemTappedEventArgsConverter.shared.cs:31]:
```suggestion
        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
            => value is ItemTappedEventArgs itemTappedEventArgs
                ? itemTappedEventArgs.Item
                : throw new ArgumentException("Expected value to be of type ItemTappedEventArgs", nameof(value));
                
        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture) 
            => throw new NotImplementedException();
```

Comment (almirvuk) [reply XamarinCommunityToolkit/Converters/ItemTappedEventArgsConverter.shared.cs]: Ok, makes sense I have noticed a similar coding style in other converters. Thanks!

Comment (AndreiMisiukevich) [reply XamarinCommunityToolkit/Converters/ItemTappedEventArgsConverter.shared.cs]: 1) Usually, we avoid braces in **IF/ELSE/WHILE etc.** statements if there is only one line under them.
2) Also **=>** in methods should be started from the new line.

Comment (almirvuk) [reply XamarinCommunityToolkit/Converters/ItemTappedEventArgsConverter.shared.cs]: I will keep that in mind for the next contributions :) Thank you!

**Comment by almirvuk** [comment]
Ok, thanks I will, just a small question. At this moment, there is no gallery page for Converters is it ok that I implement it similar to the Behaviour's one? @jfversluis

**Comment by jfversluis** [comment]
Do whatever you think will make an awesome sample app 😄

**Comment by almirvuk** [comment]
ConverterGallery Page and other components are added with samples for my converter. My converter implementation sample is not working due to bug which I reported [here](https://github.com/xamarin/XamarinCommunityToolkit/issues/158).

**Comment by almirvuk** [comment]
The solution builds and running tests locally is working as expected. Can I get some more info about failing checks in order to fix them? Thanks! @jfversluis

**Comment by jfversluis** [comment]
The build status should be public... can you see the output/errors? 

Seems unit tests needs some updating in code :)

**Comment by almirvuk** [comment]
##[error]Error: The process 'C:\Program Files\dotnet\dotnet.exe' failed with exit code 1
##[error]Dotnet command failed with non-zero exit code on the following projects : d:\a\1\s\XamarinCommunityToolkit.UnitTests\XamarinCommunityToolkit.UnitTests.csproj

These are that I can from Azure DevOps pipeline logs.

**Comment by jfversluis** [comment]
If you click through you can see more, right?

> Converters\ItemTappedEventArgsConverter_Tests.cs(16,36): error CS1729: 'ItemTappedEventArgs' does not contain a constructor that takes 3 arguments [d:\a\1\s\XamarinCommunityToolkit.UnitTests\XamarinCommunityToolkit.UnitTests.csproj]

**Comment by almirvuk** [comment]
That is weird in VS for Mac if I use the one with two I get "ItemTappedEventArgs.ItemTappedEventArgs(object, object)' is obsolete: 'Please use the constructor that reports the items index"... I will now commit these changes and see if the test is going to work now.

**Comment by almirvuk** [comment]
@jfversluis Oh I see it now... thank you for the help with an error I was not looking at right place 😄

**Comment by jfversluis** [comment]
This needs #159 first

**Pull request merged**